auth/auth-model.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../database/dbConfig")
+
+module.exports = {
```

Comment:
Keep in mind that having the exports at the top like this _only_ works when the exports are functions. If you have `const foo = () => {}` or `const foo = 5` down below and put them in this object, their exports would be `undefined`.